src.ts/agent/scanner.ts

```diff
@@ -15,20 +15,36 @@
  */
 
 import * as crypto from 'crypto';
-import * as findit from 'findit2';
+
+import * as events from 'events';
+const _findit = require('findit2');
```

Comment:
nit: would the following work?

```js
const findit: (dir: string) => events.EventEmitter = require('findit2');
```
If so, can you use this pattern elsewhere as well?